sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -39,6 +41,7 @@ DistortionCorrectorComponent::DistortionCorrectorComponent(const rclcpp::NodeOpt
   // Parameter
   time_stamp_field_name_ = declare_parameter("time_stamp_field_name", "time_stamp");
   use_imu_ = declare_parameter("use_imu", true);
+  use_3d_distortion_correction_ = declare_parameter("use_3d_distortion_correction", false);
```

Comment:
```suggestion
  use_3d_distortion_correction_ = declare_parameter("use_3d_distortion_correction");
```

We shouldn't initialize the parameters here.

I see that the other params are initialized but the correct way is to init them with the param files.

Reply:
See https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/parameters/#declare-parameter-function for more details.